### PR TITLE
[5.2] Add the ability to use continue and break in the loops

### PR DESCRIPTION
**Pull request by JavaDevl**
Laravel have lots of features, I noticed it doesn't have the ability to use the continue and break so I decided to add it to the framework and eventually contribute

### CODE REVIEW

File: tests/View/ViewBladeCompilerTest.php [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 use Mockery as m;
 use Illuminate\View\Compilers\BladeCompiler;
-
```
Comment (GrahamCampbell) [inline tests/View/ViewBladeCompilerTest.php]: revert please

**Comment by JavaDevl** [comment]
how to test styleCi on my local system ?

**Comment by GrahamCampbell** [comment]
There is no need to attempt to get StyleCI to pass locally. That's the beauty of it. If a PR is merged that's not passing on StyleCI, StyleCI automatically merges the fixes itself into laravel's code base.

**Comment by GrahamCampbell** [comment]
Nice feature. :+1:

**Comment by JavaDevl** [comment]
Glad it helped the community )

**Pull request merged**